### ISSUE TITLE
[Bug] accentColors patches in storepage although the option is not on

### ISSUE THREAD

**Issue by SpaceEnergy** [shdwmtr/millennium#159]
### Before Reporting

- [X] I found no existing issues matching my bug
- [X] My issue is not caused by a theme
- [X] My issue is not caused by a plugin

### Describe the Bug

the accentColors.css file is patched into the store page (as far as i have seen only on the homepage) and therefore overwrites the correct accent colors
![{61D93123-928F-43B2-AAB9-92AB57441476}](https://github.com/user-attachments/assets/c910dc58-ca90-4da8-9c4f-15538248af3e)
![{A3B31866-ED6F-417A-AA00-C8DB353AB4CE}](https://github.com/user-attachments/assets/f5142026-d071-4fac-b499-c59c824a1779)
![{8EDF1F82-EFBD-46A1-B174-3752D89C9C12}](https://github.com/user-attachments/assets/58a12781-7b94-4f4d-b0a4-5d0f084a6958)

### Expected Behavior

that the file is not patched if the option is not on

### Steps To Reproduce

1. idk

### Operating System

Windows

**Comment by shdwmtr** [comment]
Does this persist after a restart?

**Comment by SpaceEnergy** [comment]
yes, it's been like this for a few days now (how long really i can't say)

**Comment by yurisuika** [comment]
I am not quite sure if this is exactly related to the issue you are having, but here is something weird that probably started happening around the same time that I only figured out a week or so ago.

![Image](https://github.com/user-attachments/assets/d00b3bdb-2a22-4eb9-b83b-60c704dfadb6)

So normally my current user context menu should have the grey text for the description, but it started to become white-ish and I could not figure out what was overriding it.

![Image](https://github.com/user-attachments/assets/3d704415-784c-40cc-89e1-a10e6c41976f)

Well guess what I found out, my one variable is being overridden and used in the Steam css files.

![Image](https://github.com/user-attachments/assets/c0b6908d-baf5-4e81-ad4e-57d1797e405f)

So essentially when that variable got called in the context menu, it was doing `rgb(rgba(211, 213, 217, 0.5))` which removes the opacity of course and looks pretty white. So I found the issue and could remedy it, but what the hell is going on?

It was also found here in something about remote play in the library.

![Image](https://github.com/user-attachments/assets/613e1670-36ea-4ece-827c-687fc5d21303)

My variable of the same name doesn't have the RGB function in it. This seems to still occur when no skin is active.

But it is such a specific variable name that just seems too random. It basically takes in the option for light or dark mode and falls back to the default dark mode value (so retains support for SFP as well).

![Image](https://github.com/user-attachments/assets/bffc3338-fe58-4a71-83cf-b99121610b21)

In `library.css` there are no other results that come up for `--color-text-` or even `--color-`, only in `chunk~*.css` are there a handful of entries for `--color-progress-bar-background` and `--color-progress-bar-foreground`.

Is Valve screwing with skinners?

**Comment by shdwmtr** [comment]
Could you not just rename the color variable?

**Comment by yurisuika** [comment]
Well my solution was to just override it for the same classes. But it's very weird isn't it?

Anyhow, I have no clue if that has anything to do with OP but I thought I'd mention it.

**Comment by Plaer1** [comment]
I am having a similar issue but for the community pages (and it only happens when the skin.json has a conditions section, i'm pretty sure it's just that the conditional check is broken for the community pages):

![Image](https://github.com/user-attachments/assets/cf87c81d-afd1-4744-b9df-ddb11b19eaa7)

Removing this from the skin.json:
`    "Conditions": {
        "Use Custom Accent Color?": {
            "default": "no",
            "description": "Uses the custom Accent color feature(s) of the theme",
            "values": {
                "no": {
                    "TargetJs": {
                        "affects": [".*"],  "src": "js/utilities/na.js"
                    }
                },
                "yes": {
                    "TargetJs": {
                        "affects": [".*"], "src": "js/utilities/systemAccentColors.js" 
                    }
                }
            }
        }
    },`
returns my skin to normal:

![Image](https://github.com/user-attachments/assets/21c9a544-113f-4750-8848-30009eef7a9f)
